akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -57,7 +57,7 @@ private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_])
    */
   private var lastWasNoMessage = false
 
-  private var lastMessage: Option[M] = None
+  private var lastMessage: OptionVal[M] = OptionVal.none
```

Comment:
`OptionVal` is intended for places where extreme performance is needed, when we don't want to allocate a new instance for `Some`. That can't be a requirement for the testkit so we should use standard `Option`, unless there is another reason I have missed here?

Reply:
I'm happy with `Option` if you are.

Reply:
Oh, wait, here I'm switching from `Option` to `OptionVal` (not replacing `null` usage, that's another change).  If it's not needed I'm happy to drop it.